README.md

```diff
@@ -1,9 +1,22 @@
 # Bot Sample
 
-A sample bot using the [Microsoft Bot Framework](https://dev.botframework.com) that demonstrates one way to automate bot testing.
+A clear and simple base case bot using the [Microsoft Bot Framework](https://dev.botframework.com) that demonstrates:
+
+* One way to automate bot testing 
+* How to make a bot deployable to Azure Functions, AWS Lambda (via serverless) or restify
+* LUIS.ai integration
+
+This can be useful to those looking to get started quickly with either or all of the three points above. 
+
+To learn how to setup continuous deployment of this bot see: [Continuous deployment for Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/functions-continuous-deployment)
+
+To learn how to setup the necessary environment variables, see: [How to configure Azure Function app settings](https://docs.microsoft.com/en-us/azure/azure-functions/functions-how-to-use-azure-function-app-settings).  
+
+To get the dependent botbuilder package, run npm install within site\wwwroot\[FUNCTION_DIR]
```

Comment:
In which cases there is a need to manually run `npm install` within the App Service?

I tend to set the deployment to happen from a Git repository and in that case, the default behavior is that during deployment, the package.json is looked for for dependencies and if found, `npm install` is automatically run to install them.

I am just wondering should this sentence in the readme be augmented with a note elaborating when this is needed? What do you think?